pytorch_lightning/loggers/comet.py

```diff
@@ -120,7 +122,7 @@ def __init__(self, api_key=None, save_dir=None, workspace=None,
                 logger.exception("Failed to set experiment name for comet.ml logger")
 
     @property
-    def experiment(self):
+    def experiment(self) -> Union[CometOfflineExperiment, CometExistingExperiment, CometExperiment]:
```

Comment:
is there and parent class to these three?

Reply:
Yep, CometBaseExperiment :) good catch!